plugins/inputs/slurm/sample.conf.in

```diff
@@ -0,0 +1,25 @@
+# Gather SLURM metrics
+[[inputs.slurm]]
+  ## Slurmrestd URL. Both http and https can be used as schemas.
+  url = "http://127.0.0.1:6820"
+
+  ## Credentials for JWT-based authentication.
+  # username = "foo"
+  # token = "topSecret"
+
+  ## Enabled endpoints
+  ## List of endpoints a user can acquire data from. Available
+  ## values are: diag, jobs, nodes, partitions, reservations.
+  ## Please note incorrect endpoints will be silently ignore and
+  ## that endpoint names are case insensitive.
```

Comment:
I would actually check the values as suggested in `Init()` and do
```suggestion
  ## List of endpoints a user can acquire data from
  ## Available values are: diag, jobs, nodes, partitions, reservations.
```

Reply:
Thanks a ton for the reminder! That's been done on e7341403442c7e4a21992983a2d2d634b52a9c17.